googleapiclient/http.py

```diff
@@ -38,11 +38,19 @@
 import os
 import random
 import socket
-import ssl
 import sys
 import time
 import uuid
 
+
+try:
```

Comment:
Please add `TODO(issue 221): Remove this conditional import jibbajabba.` immediately above this line.